metadata.tex

```diff
@@ -388,4 +388,7 @@
 
 \include{sections/sentinels.tex}
 
+\clearpage
+
+\include{sections/Titans stronghold.tex}
```

Comment:
NIT: I would rename the file to `titans_stronghold.tex`

Reply:
It's not NIT, it's important 😆